Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ibmcom/swift-ubuntu:3.0.2
+
+RUN \
+  export DEBIAN_FRONTEND=noninteractive && \
+  apt-get update && \
+  apt-get install curl && \
+  # Allow installation of Node 6: https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions \
+  curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash - && \
+  apt-get update && \
+  apt-get install nodejs && \
+  # Yarn's installation instructions did not work, so we are falling back to npm to install it
+  npm install -g yarn;
```

Comment:
I'm aware, their instructions (as you provided) didn't work - this is mentioned in the comment.

Node is required as:
- Yarn is written in JS
- Gulp is written in JS